Sources/Core/JavaIR.swift

```diff
@@ -379,7 +379,7 @@ public struct JavaIR {
             case let .integer(values):
                 let names = values
                     .map { ($0.description, $0.defaultValue) }
-                    .map { "\($0.0.uppercased())(\($0.1))" }.joined(separator: ", \n")
+                    .map { "@\(JavaAnnotation.serializedName(name: "\($0.1)").rendered) \($0.0.uppercased())(\($0.1))" }.joined(separator: ", \n")
```

Comment:
I think it may have just been an oversight that the annotation wasn't added here. On line 400, we have this annotation for string-based enums.